### ISSUE TITLE
Berrigan Shire Council

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/issues#409]
This issue has been **automatically** created by PlanningAlerts for [Berrigan Shire Council](http://www.planningalerts.org.au/authorities/berrigan)

It uses the scraper [planningalerts-scrapers/multiple_horizon](https://morph.io/planningalerts-scrapers/multiple_horizon)

Only close this issue once the authority is working again on PlanningAlerts. Otherwise a new issue will just automatically created. Also, if there is a duplicate issue close the other one in favour of this one.

**Comment by tseyiokorodudu** [comment]
The development applications tracking portal is currently unavailable. 
![image](https://user-images.githubusercontent.com/31440427/146697258-837503ea-39f6-4d07-a00d-e1fb458c681e.png)

**Comment by tseyiokorodudu** [comment]
I just noticed it is one of the councils using [the NSW planning portal](https://www.planningportal.nsw.gov.au/map).

**Comment by mlandauer** [comment]
As far as I can figure out there is very little useful information on the NSW planning portal. Everything has to go through the local councils anyway.

**Comment by mlandauer** [comment]
The "blocked by authority" label is useful here to just mark that there's a problem with the council website and that we are waiting on them to follow up. Sometimes @katska gives them a call to see if they can say what's happening.

**Comment by mlandauer** [comment]
They seem to have completely removed any DA tracking from their site. So, I think it makes sense to just disable the scraper and close this as a "can't fix"

**Comment by mlandauer** [comment]
Looking now they still don't any DA tracking listed on their site. Weird.

**Comment by mlandauer** [comment]
But the scraper is working. Go figure.
